andela/travel_tool_front
# #162159462 Create content placeholder loaders on dashboard

#### What does this PR do?
create data placeholders to represent the expected data appearance

#### Description of Task to be completed?
- Replace all the spinners on the dashboard with content placeholder loaders 

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
`ft-dashboard-content-loader-162159462`
- Install dependencies:
```yarn install```
`yarn test`
- All the tests should be passing
- Start the application:
```yarn start```

- Login to the application
- Navigate to the dashboard page on the sidebar
- You should be able to get results as shown in the screenshot below.

#### Any background context you want to provide?
2. You should have the user role of a Travel Administrator or Super Administrator to be able to see the `manage` page. To do that:
  - Download `Postico` and connect to your travela database with relevant details: Db name, password, port, user
  - Click `userRoles` table and set the `roleId` to that of a Travel Administrator(29187) or Super[]Administrator(10948) 

#### What are the relevant pivotal tracker stories?
[#162159462](https://www.pivotaltracker.com/story/show/162159462)

#### Screenshots (if appropriate)
![ezgif com-crop 4](https://user-images.githubusercontent.com/22371596/48920654-2dcc9280-eeab-11e8-94fb-31b4ecd99e4e.gif)
